bears/vimscript/VintBear.py

```diff
@@ -12,7 +13,7 @@ class VintBear:
     """
 
     LANGUAGES = {'VimScript'}
-    REQUIREMENTS = {PipRequirement('vim-vint', '0.3.10')}
+    REQUIREMENTS = {PackageRequirement('broken', 'vim-vint', '0.3.10')}
```

Comment:
though I assume you want the is_installed to fail so the hack is better for now until we make a dependency class that is never met